tests/e2e/transaction/transactionBuilder.test.ts

```diff
@@ -564,7 +548,7 @@ describe("transaction builder", () => {
         amount: 1,
       });
 
-      const senderAuthenticator = sign({ signer: alice, transaction });
+      const senderAuthenticator = alice.signTransactionWithAuthenticator(transaction);
```

Comment:
This should also be included as `[Breaking]` on changelog. Some users out there use the `transactionBuilder` directly